src/needle.ts

```diff
@@ -78,10 +78,10 @@ export async function makeRequest(payload: Payload): Promise<{ success: boolean;
   do {
     try {
       response = await needle(method, url, data, options);
-      emitter.apiRequestLog(`<= Response: ${response.statusCode} ${JSON.stringify(response.body)}`.slice(0, 399));
+      emitter.apiRequestLog(`<= Response: ${response.statusCode} ${JSON.stringify(response.body)}`);
```

Comment:
Seeing this in the console since update:
<img width="717" alt="Screenshot 2021-10-08 at 08 57 33" src="https://user-images.githubusercontent.com/2239563/136512180-ff4585dd-4d82-4337-b1c4-fe7e7f78dad6.png">

Reply:
ah, i probably forgot to limit it inside vsc. 
The idea was that we need to be able to debug this data.
And i was going to do this limiting inside vsc.

Reply:
@Arvi3d got it, thanks. I'll do it on VSC side myself 👍